kiwi/bootloader/config/base.py

```diff
@@ -100,8 +100,14 @@ def setup_disk_image_config(
         :param string kernel: kernel name
         :param string initrd: initrd name
         :param string boot_options:
```

Comment:
nitpick sorry, but this is not a string. shouldn't it be `:param dict boot_options:`?

Reply:
you are so right :) I'll adapt

Reply:
done now